core/src/software/aws/toolkits/core/credentials/AwsCredentials.kt

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.credentials
```

Comment:
Why move this, you also didn't move the test that went with them

Reply:
There's nothing that requires the IDE; and the `ConnectionSettings.toEnvironment` depends on it - made sense to pull it back. This is the same pattern as with region - the `toEnvironment` methods are in the `..core.region` package.

Will move the tests.